src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -39,6 +41,20 @@ namespace ngraph
             } // namespace error
 
             NodeVector add(const Node& node) { return op::add(node); }
+            NodeVector constant(const Node& node)
+            {
+                NodeVector output_nodes;
+                for (const auto& attribute : node.attributes())
```

Comment:
More attributes may be added to `Constant` in the future. Please select the `value` attribute by name.
Let's use a helper function for selecting attributes by name here.